packages/webdriverio/src/commands/browser.ts

```diff
@@ -29,7 +29,8 @@ export * from './browser/setCookies.js'
 export * from './browser/setTimeout.js'
 export * from './browser/setWindowSize.js'
 export * from './browser/switchWindow.js'
-export * from './browser/throttle.js'
```

Comment:
By removing this export, the command is not attached to the browser scope anymore which would cause a breaking change